src/commands/instances/moderation/NameChangeCommand.ts

```diff
@@ -120,23 +120,21 @@ class NameChangeCommand extends Command {
 }
 
 function buildReviewMessage(data: NameChangeDetails['data']): string {
-  const { isNewOnHiscores, hasNegativeGains, hoursDiff, ehpDiff, ehbDiff, oldStats, newStats } = data;
```

Comment:
you can't deconstruct an object that can be undefined

Reply:
that function shouldn't even run if .data is undefined 🤔

Reply:
i was just changing to meet the contract, could change to throw an error instead

Reply:
we know that the function doesn't run when it's null or undefined, therefor those args will never be nullable, so you could just wrap with NonNullable to tell it that

`data: NonNullable<NameChangeDetails['data']>`

Reply:
done